packages/site/cms-site/src/sitePreview/pagesRouter/legacyPagesRouterSitePreviewApiHandler.ts

```diff
@@ -1,50 +1,23 @@
 import { NextApiRequest, NextApiResponse } from "next";
 
-import { Scope, SitePreviewParams } from "../SitePreviewUtils";
+import { verifySitePreviewJwt } from "../SitePreviewUtils";
 
-type GraphQLClient = {
-    setHeader(key: string, value: string): unknown;
-    request<Response, Variables>(document: string, variables?: Variables): Promise<Response>;
-};
-
-async function legacyPagesRouterSitePreviewApiHandler(req: NextApiRequest, res: NextApiResponse, graphQLClient: GraphQLClient) {
+async function legacyPagesRouterSitePreviewApiHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+    _graphQLClient: unknown /* deprecated: remove argument in v8 */,
```

Comment:
We should use a JSDoc deprecation here, then devs can see this in their IDE

Reply:
Seems like it isn't supported. Didn't know that. Then we can keep it like it is.